### ISSUE TITLE
Arch Linux requires different curl

### ISSUE THREAD

**Issue by benwaffle** [alexlarsson/spotify-app#32]
On Arch, `/usr/bin/spotify` does `LD_PRELOAD=libcurl.so.3`

libcurl.so.3 is provided by a libcurl-compat package, which is built without versioned symbols

**Comment by alexlarsson** [comment]
I don't understand what you mean by this. Spotify is not using the host libraries. The libcurl its using is from the runtime, which is libcurl.so.4. Do you have a specific problem?

**Comment by benwaffle** [comment]
yes, spotify (flatpacked) tells me 'no internet connection detected'

**Comment by alexlarsson** [comment]
That is very weird. Flatpaked apps should never see the host libcurl.
